other/upgrade_1-1.sql

```diff
@@ -1009,8 +1009,8 @@ if (!$has_attachmentType_column)
 	$ID_MSG = array();
 	while ($row = smf_mysql_fetch_assoc($request))
 	{
-		$clean_name = strtr($row['filename'], '������������������������������������������������������������', 'SZszYAAAAAACEEEEIIIINOOOOOOUUUUYaaaaaaceeeeiiiinoooooouuuuyy');
-		$clean_name = strtr($clean_name, array('�' => 'TH', '�' => 'th', '�' => 'DH', '�' => 'dh', '�' => 'ss', '�' => 'OE', '�' => 'oe', '�' => 'AE', '�' => 'ae', '�' => 'u'));
+		$clean_name = strtr($row['filename'], '������������������������������������������������������������', 'SZszYAAAAAACEEEEIIIINOOOOOOUUUUYaaaaaaceeeeiiiinoooooouuuuyy');
```

Comment:
Ugh... Phpstorm, what have you done. Was it the file encoding? :/

-1 for you, +1 for sublime.

Reply:
Apparently the conversion itself got borked, rather than the format itself, so I left it as UTF-8 and just fixed the characters. If we really want it on windows format for some reason I can change it back.